### PR TITLE
bugfix: pin nemo-text-process to fix Chinese normalizer error.

### PR DESCRIPTION
**Pull request by XuesongYang**
# What does this PR do ?

Related to [bug report](https://nvbugswb.nvidia.com/NvBugs5/SWBug.aspx?bugid=4312419&cmtNo=), 

Chinese normalizer outputted wrong normalized text for the input, '双辽境内除东辽河、西辽河等**5**条河流', where we expect '双辽境内除东辽河、西辽河等**五**条河流'.

**Evidence**

- On the latest version, i.e. nemo-text-processing=0.2.0rc0, it always normalized it as '双辽境内除东辽河、西辽河等5条河流';

- On the previous version, i.e. nemo-text-processing=0.1.8rc0, it normalized it as expected text, '双辽境内除东辽河、西辽河等五条河流'.

**Detailed Log**

- 0.2.0rc0
```
In [1]: from nemo_text_processing.text_normalization.normalize import Normalizer

In [2]: normalizer = Normalizer(lang='zh', input_case='cased', overwrite_cache=True, cache_dir="./cache_dir")

In [3]: text = '双辽境内除东辽河、西辽河等5条河流'

In [4]: normalizer.normalize(text)
Out[4]: '双辽境内除东辽河、西辽河等5条河流'
```
- 0.1.8rc0
```
Downloading nemo_text_processing-0.1.8rc0-py3-none-any.whl (2.3 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 MB 12.5 MB/s eta 0:00:00
Installing collected packages: nemo-text-processing
  Attempting uninstall: nemo-text-processing
    Found existing installation: nemo-text-processing 0.2.0rc0
    Uninstalling nemo-text-processing-0.2.0rc0:
      Successfully uninstalled nemo-text-processing-0.2.0rc0
Successfully installed nemo-text-processing-0.1.8rc0


In [1]: from nemo_text_processing.text_normalization.normalize import Normalizer

In [2]: normalizer = Normalizer(lang='zh', input_case='cased', overwrite_cache=True, cache_dir="./cache_dir")

In [3]: text = '双辽境内除东辽河、西辽河等5条河流'

In [4]: normalizer.normalize(text)
Out[4]: '双辽境内除东辽河、西辽河等五条河流'
```
**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: nemo/collections/tts/models/fastpitch.py [hunk 1]
```diff
@@ -217,6 +217,12 @@ def _setup_normalizer(self, cfg):
                 raise ImportError(
                     "`nemo_text_processing` not installed, see https://github.com/NVIDIA/NeMo-text-processing for more details"
                 )
+            # fmt: off
+            import pkg_resources
+            # safeguard to the version 0.1.8rc0 to avoid unexpected normalized outputs for Chinese. Will remove the guard once
+            # next-text-processing new release fixes the bug. Tracking bugfix in https://github.com/NVIDIA/NeMo-text-processing/issues/109.
+            pkg_resources.require("nemo_text_processing==0.1.8rc0")
```
Comment (blisc) [inline nemo/collections/tts/models/fastpitch.py:224]: Does this raise an error if the version is incorrect? If so, can you print an appropriate error message for why this is pinned?

Comment (ekmb) [inline nemo/collections/tts/models/fastpitch.py:224]: have you seen https://setuptools.pypa.io/en/latest/pkg_resources.html:
```
 Users should refrain from new usage of pkg_resources and should work to port to importlib-based solutions.
```
also maybe add a better message for users + installation instructions?

Comment (XuesongYang) [reply nemo/collections/tts/models/fastpitch.py:224]: yes, it does,
```
ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_from_pretrained_ctc_model - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_from_pretrained_rnnt_model - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_from_asr_config - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_save_restore - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_save_restore_asr - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_from_pretrained_ctc_model_fused_bn - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/tts/test_tts_exportables.py::TestExportable::test_FastPitchModel_export_to_onnx - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))
```

Comment (XuesongYang) [reply nemo/collections/tts/models/fastpitch.py:224]: good catch! we are thinking of just pin `nemo-text-process<=0.1.8rc0` to simplify the temporal solution

**Review by blisc — APPROVED** [review]

**Pull request merged**